std_msgs/Header header

uint8 predicate
uint8[] state